### ISSUE TITLE
Question about a specific part

### ISSUE THREAD

**Issue by GBenakidis** [Augustin-FL/building-pfsense-iso-from-source#55]
Hi,
Sorry for asking this question here but I couldn't reach you somewhere else. I'm trying to download packages like node.js and gcc on my pfSense VM. The part that you say "Then you need to configure nginx for PKG hosting and poudriere monitoring:", with the code below will allow me to download them successfully? If not, can you help me with that?

Best regards

**Comment by Augustin-FL** [comment]
Hi,
Yes, you can build a custom ISO with packages like gcc /  nodejs embedded.

You can also build a custom ISO without these packages, setup poudriere to build these packages afterwards, and install the packages manually on request

That being said, I don't think you need a custom ISO for this usecase. I would recommend to install the packages manually on an official pfsense instead?
`pkg install gcc` in a terminal should do the job.

Did you have a look to  https://docs.netgate.com/pfsense/en/latest/recipes/freebsd-pkg-repo.html#installing-packages ?

**Comment by GBenakidis** [comment]
Thank you for the speedy answer! 
I solved the problem. 

I didn't use custom ISO or anything like that. The main problem from the beginning, was the inability to use the `pkg` command for neither gcc nor nodejs. So I found this guy that enables the command https://gist.github.com/avin/6ffc3a5f4e4971a739c2fc03d9df2d3f. It's kinda like the link you sent, which I wasn't aware of. 
Thanks a lot and have a nice day!
